x11-misc/j4-dmenu-desktop/j4-dmenu-desktop-2.17.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+MY_PV="r${PV}"
+MY_P="${PN}-${MY_PV}"
```

Comment:
Apart from `SRC_URI` where it may as well be `${P}` instead, this is only used to set `S`. So you can drop it and simply assemble `S` by use of `${MY_PV}` instead.

Reply:
Good catch, done.